Front-End/challenge2/script.js

```diff
@@ -1,7 +1,58 @@
+/* eslint-disable default-case */
 /*
 TO-DO:
 
 - Modify this file only
 - The calculator should be completely functional
 
 */
+const allButtons = document.querySelectorAll('button');
+const display = document.querySelector('#display');
+display.textContent = '';
+
+let operatorA; let operatorB; let operation;
+
+function clean() {
+	display.textContent = '';
+}
+
+function reset() {
+	display.textContent = '';
+	operatorA = 0;
+	operatorB = 0;
+	operation = 0;
+}
+
+function resolve() {
```

Comment:
This solution is "almost there". There are a few cases still not working. E.g:

- `2 + 2 + 2` should return `6`. Your calculator returns `4`
- `1 + 2 + 3 * 2` should return `12`. Your calculator returns `6`